### ISSUE TITLE
Any plan to push code any time? Would like to contribute.

### ISSUE THREAD

**Issue by Balasnest** [vinitraj10/react-native-camerax#1]
Any plan to push code any time? I would like to contribute. Thanks!

**Comment by Balasnest** [comment]
@vinitraj10 Hi I am also developing similar one. But facing issue in getting camera preview. Its working fine in activity. Could you please help me out.

https://stackoverflow.com/questions/64789211/camerax-view-in-react-native-unable-to-configure-camera

**Comment by vinitraj10** [comment]
Hi @Balasnest , sorry for the late reply, was not finding any time to work on this, I remember was facing some hard time inflating camera-x as a react-native component, so left working on it, but would probably pick it up once again, whats up with u did you find any success with it yet??

**Comment by Balasnest** [comment]
@vinitraj10 Yeah. I had developed with cameraX + openCV through bridging after a long-pending issue on the preview (above one). But later at the final stage, faced one major issue on **FPS** drops in most of the android devices. It was giving in the range of 5-15 fps which is too low for the analysis.

So please do check for benchmark results regarding this in my [SO](https://stackoverflow.com/questions/65358326/js-ui-is-blocked-while-processing-image-analysis-of-camerax) post before you start work on this.

As of now, I moved into native for android due to this perf issue. I will start the same for iOS as well. Let see how the benchmark comes for iPhone.

Also, the entire OpenCV size is **9MB** for android (in release build). It's huge if we are consuming only 3% from SDK. (eg: highgui and mat module).

Let me know your experience and thoughts.

**Comment by mamahd12** [comment]
Hi @Balasnest, I am deleloping a react native view using native camera android X view for realtime face detection, and I can not show Preview camera, can you give me a guideline for this?
